index.html

```diff
@@ -97,9 +97,8 @@ <h2>
         <li>A payment request event type {{PaymentRequestEvent}}). A <a>payment
         handler</a> is an event handler for the {{PaymentRequestEvent}}.
         </li>
-        <li>An extension to the service worker registration interface
-        ({{PaymentManager}} to manage the definition, display, and user
-        selection of {{PaymentInstrument}}s.
+        <li>An extension to the <a>service worker registration</a> interface
+        ({{PaymentManager}} to manage properties of payment handlers.
```

Comment:
```suggestion
        ({{PaymentManager}}) to manage properties of payment handlers.
```

Or alternatively, remove the leading `(` and add commas instead

Reply:
I seem to have a paren match problem 😅